AnnoyancesFilter/sections/popups.txt

```diff
@@ -98,6 +98,9 @@ sodapdf.com##.be-tooltip
 !
 ! Specific sites
 !
+downfile.site#$##myModal { display: none !important; }
+downfile.site#$#.modal-backdrop { display: none !important; }
+downfile.site#$#body { overflow: visible !important; padding-right: 0 !important; }
```

Comment:
why `padding-right: 0 !important;`?

Reply:
I guess it's because of this. But this window contains ads, so you have to move it to the Base - `general_extensions.txt`
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/112434891-13de3b80-8d55-11eb-87c8-c5be700151ae.png)

</details><br/>

Reply:
Okay.